openapi/src/main/resources/templates/openworld-sdk/openworld/client.mustache

```diff
@@ -112,9 +118,22 @@ import java.util.UUID
     */
     @JvmOverloads
     fun {{operationId}}({{>apiParamsDecleration}}) : {{{returnType}}}{{^returnType}}Void?{{/returnType}} {
+        return {{operationId}}WithResponse({{>apiParamsPassed}}){{#returnType}}.body{{/returnType}}
```

Comment:
Please move the common parts to `api` file.

Reply:
Done ✅

Reply:
Thanks